.travis.yml

```diff
@@ -23,3 +24,6 @@ after_success:
 branches:
   only:
     - master
+env:
+  global:
+  - GH_REF: github.com/ta2edchimp/opt-cli
```

Comment:
`GH_TOKEN` needs to be added, please refer [Encrypted-Variables](https://docs.travis-ci.com/user/environment-variables/#Encrypted-Variables) for details.